1-Basic/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs

```diff
@@ -35,9 +35,14 @@ async void OnSignInSignOut(object sender, EventArgs e)
                     {
                         try
                         {
+                            // Hide the privacy prompt
```

Comment:
can you add a lnk to the wiki or doc where it explains what this is? Maybe it should be false by default...